src/Exceptions/InvalidStatusCodeException.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Logiek\Http\Exceptions;
+
+use Exception;
+use Throwable;
+
+class InvalidStatusCodeException extends Exception implements StatusCodeException
+{
+    public function __construct($statusCode = null, $code = 0, Throwable $previous = null)
+    {
+        $message = 'The given HTTP status code doesn\'t exist.';
```

Comment:
I feel like we don't have to specify the status code here since the user can find the correspondent code using the stack trace.